apps/discord-bot/internal/command/addcmdaliascommand.go

```diff
@@ -0,0 +1,42 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/senchabot-opensource/monorepo/apps/discord-bot/internal/db"
+	"github.com/senchabot-opensource/monorepo/packages/gosenchabot"
+)
+
+func (c *commands) AddCmdAliasCommand(ctx context.Context, s *discordgo.Session, i *discordgo.InteractionCreate, db db.MySQL) {
+	options := i.ApplicationCommandData().Options
+
```

Comment:
There should be length checks to avoid panic.

Reply:
With Discord's application command metadata, we can be sure of the length of the options on the Discord side.